subiquitycore/ui/views/network.py

```diff
@@ -409,9 +431,31 @@ def _create_bond(self, sender=None):
         stretchy.attach_context(self.controller.context.child("add_bond"))
         self.show_stretchy_overlay(stretchy)
 
+    def update_for_wlan_support_install_state(self, state):
+        if state == "NOT_NEEDED":
+            return
+        if self.error_showing:
+            b = 2
```

Comment:
What about instead of these b/i variables, we define some start and end points?  I think that would scan better.  What do you think?  Also I had no idea I could define a slice ahead of time, really cool.

Reply:
yeah that would be clearer, I'll change this too.